.github/workflows/server-test.yml

```diff
@@ -63,13 +63,15 @@ jobs:
       - name: Clippy
         run: |
           cd scheduler
-          cargo clippy --all -- --deny "warnings"
+          cargo clippy --all
+          # To re-enable
+          # cargo clippy --all -- --deny "warnings"
 
-      - name: Unused dependencies
-        run: |
-          cd scheduler
-          cargo install cargo-udeps --locked
-          cargo +nightly udeps --all-targets
+      # - name: Unused dependencies
```

Comment:
How come?

Reply:
Ah yes good point

It's pretty slow in the CI for the moment (6min for that in the total 8min), so I commented it out for now.
I would like to have it enabled, but maybe just keep it for master or something 🤔

Reply:
Just leave a comment to that effect.